tests/replay_rtgsfit.c

```diff
@@ -142,8 +191,16 @@ int main(int argc, char *argv[]) {
     errorExit("test_st40pcs_mds: Cannot allocate memory.");
   }
   char channel_num[3];
-  FILE *fd;
 
+  float *pcs_dec_data = (float *)calloc(pcs1_dec_len + pcs2_dec_len,
+    sizeof(float));
+  memcpy(pcs_dec_data, pcs1_dec_data, pcs1_dec_len * sizeof(float));
+  memcpy(&pcs_dec_data[pcs1_dec_len], pcs2_dec_data,
+     (pcs2_dec_len) * sizeof(float));
+  char str[100];
+  float t_step = (pcs1_dec_time[dec_N_iter - 1] - pcs1_dec_time[0]) / dec_N_iter;
+  /* t_start is referenced to plasma breakdown */
+  long int t_start_idx = (long int) ((t_start - pcs1_dec_time[0]) / t_step);
   long *t10 = calloc((size_t)N_iter, sizeof(long));
```

Comment:
size_t vs long?

Reply:
size_t is returned type value from from sizeof operation